packages/utils/src/formidlingUtils.tsx

```diff
@@ -39,28 +38,24 @@ export function lagVisningsnavnForMottaker(
   return mottakerId;
 }
 
-function lesTilgjengeligeVedtaksbrev(tilgjengeligeVedtaksbrev: TilgjengeligeVedtaksbrev): Array<string> {
-  if (tilgjengeligeVedtaksbrev?.vedtaksbrevmaler) {
-    return Object.keys(tilgjengeligeVedtaksbrev.vedtaksbrevmaler);
-  }
-  // tilgjengeligeVedtaksbrev.vedtaksbrev kan fjernes når vedtaksbrevmaler i formidling og dokumentdata er prodsatt
-  return tilgjengeligeVedtaksbrev?.vedtaksbrev ?? [];
+function vedtaksbrevmaler(tilgjengeligeVedtaksbrev: TilgjengeligeVedtaksbrev) {
+  return tilgjengeligeVedtaksbrev ? Object.keys(tilgjengeligeVedtaksbrev?.vedtaksbrevmaler) : [];
```

Comment:
Ser ikke helt nytten med optional chaining her. Dessuten vil Object.keys kaste en feil hvis den kjøres med et argument som er undefined og ikke et objekt.

Reply:
kanskje heller ha 

```suggestion
  return tilgjengeligeVedtaksbrev?.vedtaksbrevmaler ? Object.keys(tilgjengeligeVedtaksbrev.vedtaksbrevmaler) : [];
```
?